exercises/armstrong-numbers/description.md

```diff
@@ -0,0 +1,9 @@
+An [Armstrong number](https://en.wikipedia.org/wiki/Narcissistic_number) is a number that is the sum of its own digits each raised to the power of the number of digits.
+
+For example:
+9 is an Armstrong number, because 9 = 9^1 = 9
+10 is *not* an Armstrong number, because 10 != 1^2 + 0^2 = 2
+153 is an Armstrong number, because: 153 = 1^3 + 5^3 + 3^3 = 1 + 125 + 27 = 153
+154 is *not* an Armstrong number, because: 154 != 1^3 + 5^3 + 4^3 = 1 + 125 + 64 = 190
+
+Implement a function, that returns True if the number is an Armstrong number and False otherwise.
```

Comment:
There was a suggestion at some point for a better wording for this line.

Reply:
If it is that important to you, I would ask you to be more specific here.

Reply:
I think I was thinking of this: https://github.com/exercism/problem-specifications/pull/1018#discussion_r152147164

But can see now that didn't refer specifically to this text in the description.

"Write some code to determine whether a number is an Armstrong number."

A lot of different programming languages that all have their own syntaxes and conventions, so being as generic as possible in the common description which is used by all of them is helpful. Language tracks can add their own specific details if required.

See also: https://github.com/exercism/problem-specifications/issues/321

Reply:
Fixed